src/routes/auth/detail-product/+page.svelte

```diff
@@ -0,0 +1,340 @@
+<script lang="ts">
```

Comment:
auth folder does not belong to product details, 
Need to put those files in (app)/products folder